src/fontra/views/editor/editor.js

```diff
@@ -1418,11 +1456,14 @@ export class EditorController {
           };
     }
 
-    const { point: pointIndices, component: componentIndices } = parseSelection(
-      this.sceneController.selection
-    );
+    const {
+      point: pointIndices,
+      component: componentIndices,
+      anchor: anchorSelection,
```

Comment:
Please rename this variable to `anchorIndices`, in symmetry with the `componentIndices` right above it.